![Banner](https://pbs.twimg.com/profile_banners/1419589106/1589310562/1500x500)

I utilize computational and experimental methods to explore the epigenomic interface between neurodevelopment and environment. My postdoctoral research has been funded by both a CIHR Postdoctoral Fellowship award and a [CIHR Banting Postdoctoral Fellowship](http://banting.fellowships-bourses.gc.ca/en/2018-2019-eng.html) award. When not working away on my research I contribute to [EpiGenie](http://epigenie.com) as the [Epigenetics Editor & Writer](https://epigenie.com/who-is-epigenie/). 

Most of my computational efforts are related to [DMRichR](https://github.com/ben-laufer/DMRichR) and [CpG_Me](https://github.com/ben-laufer/CpG_Me), which enable the analysis of low-pass whole genome bisulfite sequencing. 
The following publications demonstrate their potential: 

1) Laufer BI, Hwang H, Jianu JM, Mordaunt CE, Korf IF, Hertz-Picciotto I, LaSalle JM. Low-Pass Whole Genome Bisulfite Sequencing of Neonatal Dried Blood Spots Identifies a Role for RUNX1 in Down Syndrome DNA Methylation Profiles. *bioRxiv* preprint. **doi**: [10.1101/2020.06.18.157693](https://doi.org/10.1101/2020.06.18.157693)

2) Laufer BI, Hwang H, Vogel Ciernia A, Mordaunt CE, LaSalle JM. Whole genome bisulfite sequencing of Down syndrome brain reveals regional DNA hypermethylation and novel disease insights. *Epigenetics*, 2019. **doi**: [10.1080/15592294.2019.1609867](https://doi.org/10.1080/15592294.2019.1609867)
  
 <p align="center">
  <a href="https://twitter.com/laufer_ben"><img src="https://img.shields.io/twitter/follow/laufer_ben?style=social" alt="Twitter"></a>
  <a href="https://www.linkedin.com/in/ben-laufer-539a496a/"><img src="https://img.shields.io/badge/LinkedIn--_.svg?style=social&logo=linkedin" alt="LinkedIn"></a>
  <a href="https://scholar.google.ca/citations?user=R8_n7dgAAAAJ&hl=en"><img src="https://img.shields.io/badge/Google%20Scholar-Profile-lightgrey?logo=google" alt="Google Scholar"></a>
</p>
